app/models/concerns/foreman_ansible/host_managed_extensions.rb

```diff
@@ -9,6 +9,7 @@ def self.prepended(base)
         include ::ForemanAnsible::Concerns::JobInvocationHelper
 
         has_many :host_ansible_roles, :foreign_key => :host_id
+        accepts_nested_attributes_for :host_ansible_roles, :allow_destroy => true
         has_many :ansible_roles, :through => :host_ansible_roles,
```

Comment:
Could we order by default? :thinking: 
```suggestion
        has_many :ansible_roles, -> { order('host_ansible_roles.position ASC') }, :through => :host_ansible_roles,
```